docs/user-guide.md

```diff
@@ -115,56 +115,57 @@ module.exports = {
 };
 ```
 
-For example, if you are using the `"@typescript-eslint/parser"`, and if you want to use TypeScript in `<script>` of `.svelte`, you need to add more `parserOptions` configuration.
+For example, if you are using the `"typescript-eslint/parser"`, and if you want to use TypeScript in `<script>` of `.svelte`, you need to add more `parserOptions` configuration.
 
 ```js
-module.exports = {
-  // ...
-  extends: ['plugin:svelte/recommended'],
-  // ...
-  parser: '@typescript-eslint/parser',
-  parserOptions: {
-    // ...
-    project: 'path/to/your/tsconfig.json',
-    extraFileExtensions: ['.svelte'] // This is a required setting in `@typescript-eslint/parser` v4.24.0.
+import svelteParser from 'svelte-eslint-parser';
+import tseslint from 'typescript-eslint';
+
+export default [
+  //...
+  {
+    files: ['**/*.ts', '**/*.js', '**/*.svelte', '**/*.cjs'],
+    languageOptions: {
+      parser: tseslint.parser,
```

Comment:
I don't think we need to use `tseslint.parser` for `*.js` and `*.svelte`, I think it makes sense to specify only `parserOptions` here.

```suggestion
```

Maybe we need another section that only specifies `parser: tseslint.parser`. What do you think?